README.md

```diff
@@ -7,11 +7,11 @@ Ubuntu used by default, Alpine builds also available tagged as `-alpine`
 
 Provides base OS, patches and stable nginx for quick and easy spinup.
 
-[S6](https://github.com/just-containers/s6-overlay) process supervisor is used for `only` for zombie reaping (as PID 1), boot coordination, and termination signal translation  
+[S6](https://github.com/just-containers/s6-overlay) process supervisor is used for `only` for zombie reaping (as PID 1), boot coordination, and termination signal translation
```

Comment:
It looks like your editor clipped the whitespace, which is important in MD files